documentation/staging/content/faq/node-heating.md

```diff
@@ -12,7 +12,7 @@ One solution is to ensure that all necessary container images are available on w
 
 The operator team recommends a different solution that is based on [inter-pod affinity and anti-affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity). This solution has the advantage of both resolving the Node heating problem and of explicitly directing the Kubernetes Scheduler to spread the Pods for WebLogic Server instances from a given cluster or domain more widely across the available Nodes. Inter-pod affinity and anti-affinity are features of the Kubernetes Scheduler that allow the scheduler to choose a Node for a new Pod based on details of the Pods that are already running. For WebLogic Server use cases, the intent will often be for anti-affinity with the Pods for other WebLogic Server instances so that server instances spread over the available Nodes.
 
-To use these features, edit the Domain Custom Resource to add content to the `serverPod` element, in this case at the scope of a cluster, as shown in the following example:
+Beginning with operator version 4.0, the operator adds the following content to the `serverPod` element at the cluster scope in the Domain Custom Resource by default.
```

Comment:
Beginning with operator version 4.0, the operator adds the following content to the `serverPod` element at the cluster scope in the Domain Custom Resource by default. -> Beginning with operator version 4.0, by default, the operator adds the following content to the `serverPod` element at the cluster scope in the Domain Custom Resource.